assets/GroupPolicy/InstallPSCorePolicyDefinitions.ps1

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
```suggestion
# Copyright (c) Microsoft Corporation.
```

Sent you offline message with updated guidance.

Reply:
not blocking

Reply:
Fixed.